scripts/build-color-variants.js

```diff
@@ -480,11 +480,13 @@ function buildColorVariants(variables, config) {
       const colorValue = variables[color];
       return result += stripIndent(`
         .shadow-${color}-on-hover:hover,
-        .shadow-${color}-on-active.is-active {
+        .shadow-${color}-on-active.is-active,
+        .shadow-${color}-on-active.is-active:hover {
```

Comment:
😭 I thought the specificity was slightly higher when targeting pseudo selector. Sorry about that. will fix. 😭